src/ripper.rb

```diff
@@ -325,10 +332,12 @@ class InvalidRubyVersionError < StandardError
     REQUIRED = Gem::Version.new('2.5')
 
     def initialize
-      super(<<~ERR)
-        Ruby version #{RUBY_VERSION} not supported.
+      super(
+        <<~ERR
+                Ruby version #{RUBY_VERSION} not supported.
```

Comment:
@kddeisz I've rebased on master now and applied prettier to this file. Unfortunately this keeps getting indented on each format though.

If I run prettier again:

```
prettier --plugin=. --write src/ripper.rb
```

This gets indented:

```diff
     def initialize
       super(
         <<~ERR
-                Ruby version #{RUBY_VERSION} not supported.
+                                Ruby version #{RUBY_VERSION} not supported.
         Please upgrade to #{REQUIRED} or above.
       ERR
       )
```

Reply:
Yeah I know about this one. I rewrote it to not use a heredoc for now.